pmd-rust/src/main/java/net/sourceforge/pmd/lang/rust/ast/RustLexer.java

```diff
@@ -0,0 +1,1233 @@
+// CHECKSTYLE:OFF Suppressing checkstyle checks for generated code
+// Generated from RustLexer.g4 by ANTLR 4.13.2
+// CPD-OFF: Suppressing CPD checks for generated code
+package net.sourceforge.pmd.lang.rust.cpd;
+
+import org.antlr.v4.runtime.Lexer;
+import org.antlr.v4.runtime.CharStream;
+import org.antlr.v4.runtime.Token;
+import org.antlr.v4.runtime.TokenStream;
+import org.antlr.v4.runtime.*;
+import org.antlr.v4.runtime.atn.*;
+import org.antlr.v4.runtime.dfa.DFA;
+import org.antlr.v4.runtime.misc.*;
+
+import net.sourceforge.pmd.lang.rust.ast.RustLexerBase;
+
+@SuppressWarnings({"all", "warnings", "unchecked", "unused", "cast", "CheckReturnValue", "this-escape"})
+public class RustLexer extends RustLexerBase {
```

Comment:
why is this file here? if it's generated from the grammar during build (under `build/generated-sources/antlr4`) there is no need to have it under `src/main/java`…

Reply:
oops! that's my bad. I inlined the custom functions from the ANTLR grammar in `RustLexer.g4` instead and added a note about the modification to the README in 0f58dae1f25ae3c6b72f6dc1e070d86599c3e716. thank you! :)